### PR TITLE
Examples + Moar! (woof)

### PR DESCRIPTION
**Pull request by minotaa**

### CODE REVIEW

File: .examples/discord_example.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This example uses discord.py
+
+import discord
```
Comment (RDIL) [inline .examples/discord_example.py:3]: Please add `discord.py` and `asyncio` to a `requirements.txt` file.

File: .examples/discord_example.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# This example uses discord.py
```
Comment (RDIL) [inline .examples/discord_example.py:1]:
```suggestion
# This example uses discord.py (the rewrite branch)
```

File: .examples/discord_example.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# iDoggo modules
+from iDoggo import dog1
+from iDoggo import dog2
+import iDoggo
+
+bot = commands.Bot(command_prefix="!")
+
+@bot.commands(pass_context=True)
+async def dog1(ctx):
+     doggo = iDoggo.dog1()
```
Comment (RDIL) [inline .examples/discord_example.py:16]: Can be simplified just to `await bot.say(iDoggo.dog1())`

**Review by RDIL — CHANGES_REQUESTED** [review]

File: .examples/discord_example.py (hunk 2 above)
Comment (minotaa) [reply .examples/discord_example.py:1]: It's not the rewrite branch.

Comment (RDIL) [reply .examples/discord_example.py:1]: Oh huh.

File: .examples/discord_example.py (hunk 1 above)
Comment (minotaa) [reply .examples/discord_example.py:3]: You can't install asyncio

and also we don't have to install discord.py as it is just an example

File: .examples/discord_example.py (hunk 3 above)
Comment (minotaa) [reply .examples/discord_example.py:16]: eh.

**Review by RDIL — APPROVED** [review]

File: .examples/discord_example.py (hunk 3 above)
Comment (N8python) [reply .examples/discord_example.py:16]: await is easier then setting up an async function. Either way, this all looks good.

**Review by N8python — APPROVED** [review]
Very good!

File: .examples/discord_example.py (hunk 3 above)
Comment (minotaa) [reply .examples/discord_example.py:16]: eh, i rushed it, should be fine though

Comment (N8python) [reply .examples/discord_example.py:16]: Yeah - its good. You can certainly do ore than I can with discord.

**Comment by RDIL** [comment]
Thanks for your contributions, I'm going to build and push to PyPI later.

**Comment by RDIL** [comment]
This version is now on PyPI.  Sorry for the huge delay.

**Pull request merged**